common/sgx/verifier.c

```diff
@@ -302,6 +312,91 @@ static oe_result_t _fill_with_known_claims(
         format_id,
         sizeof(*format_id)));
 
+    // SGX Report CPU SVN
+    OE_CHECK(_add_claim(
+        &claims[claims_index++],
+        OE_CLAIM_SGX_CPU_SVN,
+        sizeof(OE_CLAIM_SGX_CPU_SVN),
+        sgx_report_body->cpusvn,
+        sizeof(sgx_report_body->cpusvn)));
+
+    // SGX Exit Info Reported to an SSA Frame or Not
+    flag = !!(sgx_report_body->miscselect & SGX_MISC_FLAGS_EXIT_INFO);
+    OE_CHECK(_add_claim(
+        &claims[claims_index++],
+        OE_CLAIM_SGX_EXIT_INFO_ENABLED,
```

Comment:
The claim name is not informative. The specific bit in miscselect indicates whether page fault or general protection exception occurred inside the SGX enclave is reported to the enclave SW. It's possible that future SGX version might add new bits also related to EXINFO. As the purpose of translating the SGX miscselect bitmap to SGX-specific claims is to simplify the upper level SW developer's work, it's better to make the claim names informative. Suggest to rename the claim as `OE_CLAIM_SGX_PF_GP_EXINFO_ENABLED`.

Reply:
Thanks for the naming.
Renamed it.